editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/FusePropertySection.java

```diff
@@ -674,6 +675,23 @@ public void modifyText(ModifyEvent e) {
 			txtField.setLayoutData(createPropertyFieldLayoutData());
 			c = txtField;
 
+			// grand Children inside a list of nodes
+		} else if(camelModelElement != null && camelModelElement.getParameter(p.getName()) instanceof List){
+			//TODO: use something better than a text for a list of elements as we have the obvious limitation of not supporting the ","
+			List<String> initialValue = (List<String>)camelModelElement.getParameter(p.getName());
+			String delimiter = ",";
+			String collect = initialValue.stream().collect(Collectors.joining(delimiter));
+			Text txtField = getWidgetFactory().createText(parent, collect, SWT.SINGLE | SWT.LEFT);
+			txtField.addModifyListener(new ModifyListener() {
+				@Override
+				public void modifyText(ModifyEvent e) {
+					Text txt = (Text) e.getSource();
+					camelModelElement.setParameter(p.getName(), Arrays.asList(txt.getText().split(delimiter)));
```

Comment:
I can see both sides of this, but unless there's any example someone could come up with involving the need to have leading or trailing spaces around a value in a list of property values, then it seems like trimming those to avoid bugs would outweigh the flexibility of leaving it to the user/developer to filter them out when unintentional. I.e, it seems like their unintentional entry could be fairly common vs. their need in leading/trailing a value, which I suspect is extremely rare, if not non-existent.